compiler/x/codegen/OMRInstOpCode.hpp

```diff
@@ -490,13 +503,13 @@ class InstOpCode: public OMR::InstOpCode
                {
                supported = target->supportsFeature(OMR_FEATURE_X86_AVX512F);
 
-               if (supported && flags & X86FeatureProp_EVEX128RequiresAVX512VL)
+               if (supported && flags & X86FeatureProp_AVX512VL)
```

Comment:
I know it isn't required by the C++ language rules but can you add parentheses around the `&` expressions here and elsewhere in this PR so the operator precedence is crystal clear to human readers?  I see this is the way the code was already written, but I think it is a good time to change while you're here.